src/schema/devcards.ts

```diff
@@ -40,6 +40,26 @@ export const typeDefs = /* GraphQL */ `
     X
   }
 
+  """
+  Source for dev card frequent sources
+  """
+  type DevCardSource {
```

Comment:
Can we utilize the existing source type please? I know it'll be somewhat broken but it's better for consistency. We can always fix fact that it's broken after the hackathon